### PR TITLE
add fetchClosedOrders to braziliex

### PR DESCRIPTION
**Pull request by mauriciosl**
This adds fetchClosedOrders to the Braziliex exchange.

### CODE REVIEW

**Comment by kroitor** [comment]
Sorry, we have to correctly interpret fetchClosedOrders compared to fetchMyTrades – those are different types of data (check out a few lines below your edits: 

- https://github.com/ccxt/ccxt/pull/6655/files#diff-a405aefc0e5b86d4bb4cb429323e0268R563

The difference between closed orders and trades (or trade history) is explained here:

- https://github.com/ccxt/ccxt/wiki/Manual#how-orders-are-related-to-trades

In other words, Braziliex truly does not have `fetchClosedOrders`, but has `fetchMyTrades` instead, and this is already outlined in the `'has'` metadata and in the implementation.

Let us know if we are missing something. 

We appreciate all help and all contributions, and will be happy to see more from you in the future! Thank you!

**Comment by mauriciosl** [comment]
Hi,

Thanks for the explanation, I'm learning ccxt and trying to fix things as I go.
I will check the pointed documentation and fix Braziliex API so that we can have the closed orders too, once it's done I will submit a new PR.

**Comment by kroitor** [comment]
@mauriciosl awesome, thanks again!
